rocAL/rocAL_pybind/amd/rocAL/fn.py

```diff
@@ -0,0 +1,547 @@
+    return (cropped_image)
+
+def color_twist(*inputs, brightness=1.0, bytes_per_sample_hint=0, contrast=1.0, hue=0.0, image_type=0,
+                preserve=False, saturation=1.0, seed=-1, device=None):
+    brightness = b.CreateFloatParameter(brightness) if isinstance(
+        brightness, float) else brightness
+    bytes_per_sample_hint = bytes_per_sample_hint
+    contrast = b.CreateFloatParameter(
+        contrast) if isinstance(contrast, float) else contrast
+    hue = b.CreateFloatParameter(hue) if isinstance(hue, float) else hue
+    saturation = b.CreateFloatParameter(saturation) if isinstance(
+        saturation, float) else saturation
+    # pybind call arguments
+    kwargs_pybind = {"input_image0": inputs[0], "is_output": False,
+                     "alpha": brightness, "beta": contrast, "hue": hue, "sat": saturation}
+    color_twist_image = b.ColorTwist(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    return (color_twist_image)
+
+def uniform(*inputs,range=[-1, 1], device=None):
+    output_param = b.CreateFloatUniformRand(range[0], range[1])
+    return output_param
+
+def random_bbox_crop(*inputs,all_boxes_above_threshold = True, allow_no_crop =True, aspect_ratio = None, bbox_layout = "", bytes_per_sample_hint = 0,
+                crop_shape = None, input_shape = None, ltrb = True, num_attempts = 1 ,scaling =  None,  preserve = False, seed = -1, shape_layout = "",
+                threshold_type ="iou", thresholds = None, total_num_attempts = 0, device = None, labels = None ):
+    aspect_ratio = aspect_ratio if aspect_ratio else [1.0, 1.0]
+    if crop_shape is None:
+        crop_shape = []
+    else:
+        crop_shape = crop_shape
+    if input_shape is None:
+        input_shape = []
+    else:
+        input_shape = input_shape
+    scaling = scaling if scaling else [1.0, 1.0]
+    thresholds = thresholds if thresholds else [0.0]
+    crop_begin = []
+    if(len(crop_shape) == 0):
+        has_shape = False
+        crop_width = 0
+        crop_height = 0
+    else:
+        has_shape = True
+        crop_width = crop_shape[0]
+        crop_height = crop_shape[1]
+    scaling = b.CreateFloatUniformRand(scaling[0], scaling[1])
+    aspect_ratio = b.CreateFloatUniformRand(aspect_ratio[0], aspect_ratio[1])
+
+    # pybind call arguments
+    kwargs_pybind = {"all_boxes_above_threshold":all_boxes_above_threshold, "no_crop": allow_no_crop, "p_aspect_ratio":aspect_ratio, "has_shape":has_shape, "crop_width":crop_width, "crop_height":crop_height, "num_attemps":num_attempts, "p_scaling":scaling, "total_num_attempts":total_num_attempts }
+    random_bbox_crop = b.RandomBBoxCrop(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+
+    return (random_bbox_crop,[],[],[])
+
+def bb_flip(*inputs, bytes_per_sample_hint = 0, horizontal = 1, ltrb = False, preserve =  False,seed = -1, vertical = 0, device = None):
```

Comment:
please add comment specifying why this is a dummy node

Reply:
Done
Added the comment to tell why this is a dummy node